recipes/ckan-deploy.rb

```diff
@@ -30,9 +30,16 @@
 shared_fs_dir = "/var/shared_content/#{app['shortname']}"
 virtualenv_dir = "/usr/lib/ckan/default"
 pip = "#{virtualenv_dir}/bin/pip --cache-dir=/tmp/"
-paster = "#{virtualenv_dir}/bin/paster --plugin=#{service_name}"
+ckan_cli = "#{virtualenv_dir}/bin/ckan_cli"
 install_dir = "#{virtualenv_dir}/src/#{service_name}"
 
+cookbook_file "#{virtualenv_dir}/bin/ckan_cli" do
```

Comment:
Are we sure this is run before the other scripts are deployed/called?

Reply:
I searched for all occurrences of `paster` in the cookbook and am confident this will run first. There were no usages of paster in the 'setup' recipes, and this block runs before all the 'deploy' usages.